src/commands/bourne/import.ts

```diff
@@ -99,8 +99,8 @@ export default class Import extends SfdxCommand {
       let record = JSON.parse(original);
       if (configObject.hasRecordTypes && recordTypeRef) {
         if (recordTypeRef.hasOwnProperty(record.RecordType.DeveloperName)) {
+          record.RecordTypeId = recordTypeRef[record.RecordType.DeveloperName];
           delete record.RecordType;
-          record.RecordTypeId = recordTypeRef.developerName;
```

Comment:
recordTypeRef doesnt have this property: developerName.
The object looks like:
```
{
  "RecordTypeName": 'id...',
  "RecordTypeName": 'id...',
}
```

Reply:
Then I changed to get the recordType from the right place:
record.RecordTypeId = recordTypeRef[record.RecordType.DeveloperName];